src/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2 {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  font-style: normal;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card_rewies_info {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__img {
+  width: 160px;
+  margin: 0 auto;
+  padding-bottom: 40px;
+}
+
+.card__img-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__title {
+  text-align: left;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  margin-bottom: 4px;
+  width: 100%;
+  line-height: 18px;
+}
+
+.card__code {
+  text-align: left;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  width: 100%;
+  margin-bottom: 18px;
+  margin-top: 0;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 8px;
+  width: 100%;
+  margin-top: 24px;
+}
+
```

Comment:
Better use one direction for margin (top or bottom / left or right) to avoid margin collapse